vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -40,4 +40,10 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_groups
+    num_of_container_groups = @record.number_of(:container_groups)
+    label = "Container groups"
+    {:label => label, :image => "container_group", :value => num_of_container_groups}
```

Comment:
@abonas same here, i think this should be a link to List of Container Groups for the selected Container Node.